src/components/TranslationHelpsWrapper.js

```diff
@@ -51,26 +51,33 @@ function TranslationHelpsWrapper(props) {
   const groupId = contextId.groupId;
   const languageId = toolsSelectedGLs[currentToolName];
 
+  function getArticleFromReducer(resourceDir, article) {
```

Comment:
Would you mind adding the jsdocs documentation. Not sure what resourceDir is.

Reply:
Good point.  Not sure I do either - will look it up.

Reply:
Okay thanks